Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -34,11 +34,67 @@ public static class TextTable
         /// <summary>
         /// Build a table for given message with given padding.
         /// Padding means how many spaces will a message be wrapped with.
-        /// Table itself is made of: "+-" symbols. 
+        /// Table itself is made of: "+-" symbols.
         /// </summary>
         public static string Build(string message, int padding)
         {
-            return "";
+            if (string.IsNullOrEmpty(message) && padding == 0)
+            {
+                return "";
+            }
+
+            var longestWordInMessageSize = LongestWordInMessageSize(message);
+
+            var newHorizontalBorder = new StringBuilder();
+            newHorizontalBorder.Append(cornerChar).Append(horizontalWallChar, longestWordInMessageSize + (padding * 2)).Append(cornerChar).Append(Environment.NewLine);
+
+            var newEmptyLine = new StringBuilder();
+            newEmptyLine.Append(verticalWallChar).Append(' ', longestWordInMessageSize + (padding * 2)).Append(verticalWallChar).Append(Environment.NewLine);
+
+            var sb = new StringBuilder();
+
+            sb.Append(newHorizontalBorder);
+            for (var i = 0; i < padding; i++)
+            {
+                sb.Append(newEmptyLine);
+            }
+
+            foreach (var word in message.Split(dividerChar))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['message' is null on at least one execution path.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4930019)